modules/serverless-rn-1-20-0.adoc

```diff
@@ -0,0 +1,41 @@
+[id="serverless-rn-1-20-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.20.0
+
+[id="new-features-1-20-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.26.
+* {ServerlessProductName} now uses Knative Eventing 0.26.
+* {ServerlessProductName} now uses Kourier 0.26.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.26.
+* {ServerlessProductName} now uses Knative Kafka 0.26.
+* The `kn func` CLI plug-in now uses `func` FIXME.
+// Is `kn func` using version later than 0.19?
+
+* The Kafka broker is now available as a Technology Preview.
+
+[id="fixed-issues-1-20-0_{context}"]
+== Fixed issues
+
+[id="known-issues-1-20-0_{context}"]
+== Known issues
+
+* {ServerlessProductName} deploys kservices with the default address that uses HTTPS. When using the in-cluster sender job, the sender does not have the cluster Certificate Authority (CA) configured.
```

Comment:
```suggestion
* {ServerlessProductName} deploys Knative services with a default address that uses HTTPS. When using the in-cluster sender job, the sender does not have the cluster certificate authority (CA) configured. This causes delivery to fail, unless the cluster uses globally accepted certificates.
```

I'd do this instead and remove the next line

Reply:
tbh it's not really clear to me what any of this means - we don't previously mention things like "in-cluster sender job" anywhere in the current docs that I'm aware of. If this is `kn event` specific stuff I think it needs to be made clearer what it relates to.

Reply:
Yes. This is `kn event` internal machinery - the way it sends events to cluster internal components. The known issue is definitely about `kn event` fault, but in conjunction to how knative is configured as OpenShift Serverless distribution (HTTPS by default).